cluster/common.sh

```diff
@@ -423,6 +423,12 @@ function build-runtime-config() {
   :
 }
 
+function write-cluster-location {
```

Comment:
Should this have a comment and document "Assumed vars", as most other functions?

Reply:
Done